simde/x86/sse4.2.h

```diff
@@ -37,6 +37,232 @@ SIMDE__BEGIN_DECLS
 #  define SIMDE_X86_SSE4_2_ENABLE_NATIVE_ALIASES
 #endif
 
+#define SIMDE_SIDD_CMP_EQUAL_ANY     0
+#define SIMDE_SIDD_CMP_RANGES     1
+#define SIMDE_SIDD_CMP_EQUAL_EACH     2
+#define SIMDE_SIDD_CMP_EQUAL_ORDERED   3
+
+#if defined(SIMDE_X86_SSE4_2_ENABLE_NATIVE_ALIASES)
+#define _SIDD_CMP_EQUAL_ANY SIMDE_SIDD_CMP_EQUAL_ANY
+#define _SIDD_CMP_RANGES SIMDE_SIDD_CMP_RANGES
+#define _SIDD_CMP_EQUAL_EACH SIMDE_SIDD_CMP_EQUAL_EACH
+#define _SIDD_CMP_EQUAL_ORDERED SIMDE_SIDD_CMP_EQUAL_ORDERED
+#endif
+
+SIMDE__FUNCTION_ATTRIBUTES
+int
+simde_mm_cmpestra_8_(simde__m128i a, int la, simde__m128i b, int lb, const int imm8) {
+  const int cmp_op = imm8 & 0x06;
+  const int polarity = imm8 & 0x30;
```

Comment:
`polarity` is unused, delete it?

Reply:
It had to be used, I just missed it, I have corrected it now.